swagger.json

```diff
@@ -5200,64 +5200,115 @@
+        },
+        "DocumentBase": {
+            "type": "object",
             "properties": {
-                "driverId": {
-                    "type": "integer",
-                    "format": "int64",
-                    "description": "ID of the driver that submitted this document.",
-                    "example": 555
-                },
-                "driverCreatedAtMs": {
-                    "type": "integer",
-                    "format": "int64",
-                    "description": "The time in Unix epoch milliseconds that the document was created by the driver.",
-                    "example": 1462881998034
-                },
-                "documentType": {
-                    "type": "string",
-                    "description": "Descriptive name of this type of document.",
-                    "example": "Fuel Receipt"
-                },
                 "dispatchJobId": {
                     "type": "integer",
                     "format": "int64",
-                    "description": "ID of the Samsara dispatch job that the document was submitted for.",
+                    "description": "ID of the Samsara dispatch job that the document is submitted for.",
                     "example": 773
                 },
-                "vehicleId": {
-                    "type": "integer",
-                    "format": "int64",
-                    "description": "VehicleID of the driver at document creation.",
-                    "example": 222
-                },
                 "notes": {
                     "type": "string",
                     "description": "Notes submitted with this document.",
                     "example": "Fueled up before delivery."
-                },
-                "fields": {
-                    "description": "The fields associated with this document.",
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/DocumentFields"
-                    }
                 }
             }
         },
+        "Document": {
+            "allOf": [
+                {
+                    "type": "object",
+                    "required": [
+                        "driverId",
+                        "driverCreatedAtMs",
+                        "documentType",
+                        "fields"
+                    ],
+                    "properties": {
+                        "driverId": {
+                            "type": "integer",
+                            "format": "int64",
+                            "description": "ID of the driver that submitted this document.",
+                            "example": 555
+                        },
+                        "driverCreatedAtMs": {
+                            "type": "integer",
+                            "format": "int64",
+                            "description": "The time in Unix epoch milliseconds that the document was created by the driver.",
```

Comment:
nit: Can we change `driverId` and `driverCreatedAtMs`'s descriptions to be consistent in tense with `dispatchJobId`'s description?